tasks/integrated-benchmark/src/work_env.rs

```diff
@@ -26,17 +27,21 @@ pub struct WorkEnv {
 }
 
 impl WorkEnv {
-    const INIT_PROXY_CACHE: &str = ".init-proxy-cache";
-    const PNPM: &str = "PNPM";
+    const INIT_PROXY_CACHE: SubDir<'static> = SubDir::Static(".init-proxy-cache");
+    const PNPM: SubDir<'static> = SubDir::Static("pnpm");
 
     fn root(&self) -> &'_ Path {
         &self.root
     }
 
-    fn revisions(&self) -> impl Iterator<Item = &'_ str> + '_ {
+    fn revision_names(&self) -> impl Iterator<Item = &'_ str> + '_ {
         self.revisions.iter().map(AsRef::as_ref)
     }
 
+    fn revision_subs(&self) -> impl Iterator<Item = SubDir<'_>> + '_ {
+        self.revisions.iter().map(AsRef::as_ref).map(SubDir::PacquetRevision)
```

Comment:
can't this be refactored to
```suggestion
        self.revision_names().map(SubDir::PacquetRevision)
```